Wikipedia/Code/ArticleViewController.swift

```diff
@@ -119,6 +126,7 @@ class ArticleViewController: ViewController, HintPresenting {
         contentSizeObservation?.invalidate()
         messagingController.removeScriptMessageHandler()
         articleLoadWaitGroup = nil
+        altTextBottomSheetViewModel = nil
```

Comment:
@mazevedofs Were you seeing some memory issues without this line? I don't think it hurts to have it but I thought it might be unnecessary.

Reply:
No issues so far, but I feared I could be missing something since I had to keep the reference strong. We can remove it in future PRs for the bottom sheet.